internal/librarian/release_init.go

```diff
@@ -203,55 +203,65 @@ func (r *initRunner) runInitCommand(ctx context.Context, outputDir string) error
 	return copyGlobalAllowlist(r.librarianConfig, r.repo.GetDir(), outputDir, false)
 }
 
-// updateLibrary updates the given library in the following way:
-//
-// 1. Update the library's previous version.
-//
-// 2. Get the library's commit history in the given git repository.
-//
-// 3. Override the library version if libraryVersion is not empty.
-//
-// 4. Set the library's release trigger to true.
-func (r *initRunner) updateLibrary(library *config.LibraryState) error {
+// processLibrary wrapper to process the library for release. Helps retrieve latest commits
+// since the last release and passing the changes to updateLibrary.
+func (r *initRunner) processLibrary(library *config.LibraryState) error {
```

Comment:
This additional wrapper seems redundant.

What's the issue of getting commits inside `updateLibrary`?

Reply:
The previous work to setup `UpdateLibrary`'s test requires creating a gitrepo, configuring the commits, creating files, configuring the source roots to match the file paths, and setting up a tag to retrieve the commits (i.e. Logic inside `GetConventionalCommitsSinceLastRelease()`. 

This small wrapper allows `UpdateLibrary`'s test to take in a list of commits without all the previous setup.